dll/win32/rpcrt4/rpc_async.c

```diff
@@ -158,3 +158,47 @@ RPC_STATUS WINAPI RpcAsyncCancelCall(PRPC_ASYNC_STATE pAsync, BOOL fAbortCall)
     FIXME("(%p, %s): stub\n", pAsync, fAbortCall ? "TRUE" : "FALSE");
     return RPC_S_INVALID_ASYNC_HANDLE;
 }
+
```

Comment:
We'll want this inside `#ifdef __REACTOS__`

Reply:
Done.